src/test/java/guitests/StartupBoardLauncherTest.java

```diff
@@ -0,0 +1,57 @@
+package guitests;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.loadui.testfx.utils.FXTestUtils;
+import org.loadui.testfx.utils.TestUtils;
+import prefs.Preferences;
+import ui.BoardAutoCreator;
+import ui.TestController;
+import ui.UI;
+import ui.issuepanel.PanelControl;
+import ui.listpanel.ListPanel;
+import util.PlatformEx;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.FutureTask;
+
+import static guitests.BoardAutoCreatorTest.verifyBoard;
+
+import static org.junit.Assert.assertEquals;
+import static ui.BoardAutoCreator.SAMPLE_BOARD;
+
+public class StartupBoardLauncherTest extends UITest{
+
+    private PanelControl panelControl;
+
+    @Override
+    public void launchApp(){
+        FXTestUtils.launchApp(TestUI.class, "--startupboard=true");
+    }
+
+    @Before
+    public void init(){
+        panelControl = TestController.getUI().getPanelControl();
+    }
+
+    @Test
+    public void boardCreation_firstTimeUser_sampleBoardCreated(){
+        login("dummy", "dummy", "test", "test");
+        TestUtils.awaitCondition(() -> BoardAutoCreator.getSamplePanelDetails().size() == countPanelsShown());
```

Comment:
Sure. This line shows we are waiting for something to complete, which is not surprising in UI tests and is unlikely to confuse the reader. An unexpected logout with an incomprehensible comment is not.